neurom/features/__init__.py

```diff
@@ -142,12 +144,13 @@ def get(feature_name, obj, **kwargs):
     Arguments:
         feature_name(string): feature to extract
         obj: a morphology, a morphology population or a neurite tree
+        clear_cache (bool): If True, the cache of this feature is cleared before computation.
```

Comment:
arg is `cache`, not `clear_cache` ?

Reply:
Ups yes indeed, I changed the code but not the docstring...